code/game/objects/items/storage/fancy.dm

```diff
@@ -201,9 +201,6 @@
 	new /obj/item/clothing/mask/cigarette/emergency(src)
 	new /obj/item/tool/lighter(src)
 
-/obj/item/storage/fancy/chemrettes/update_icon_state()
-	icon_state = "[initial(icon_state)][contents.len]"
```

Comment:
This has been fixed now in #7216, also wouldn't this just make the sprite not update when you take something out of it?

Reply:
Oh im sorry, Im a beginner contributor and didn't notice it was aleready fixed.
Indeed this is to update the icon state, but it is a override of the standard `/obj/item/storage/fancy/update_icon_state()`. The shown override is ment for the namig convention of the standard cigarette pack icons which the chempacket icon naming dosen't follow so I just deleted it so the chempackets can use the standard `update_icon_state()` which works with it.

This solution in quicker but your's seems more reasonable since all cigarettes use standardized icon namig then.

Reply:
It's alright, i get what you in mind and that's still good. Keep up the good work eitherway!